### PR TITLE
fix: username suggestion chip styles

### PR DESCRIPTION
**Pull request by waheedahmed**
VAN-609

## Figma
Screenshot 2021-06-30 at 12 14 45 PM

## Before
Screenshot 2021-06-30 at 12 15 54 PM

## After
Screenshot 2021-06-30 at 1 50 40 PM

### CODE REVIEW

File: src/redesign/_style.scss [hunk 1]
```diff
@@ -532,15 +532,17 @@ select.form-control {
 }
 
 .username-suggestion {
-  padding: 2px 0.5rem;
+  padding: 1px 0.5rem;
```
Comment (shafqatfarhan) [inline src/redesign/_style.scss:535]: nit: Might need to update padding-top to 2px as Figma shows padding `padding: 2px 8px;`.

**Review by shafqatfarhan — APPROVED** [review]

**Comment by shafqatfarhan** [comment]
@waheedahmed Suggested button color is `Primary/700`

**Review by mubbsharanwar — APPROVED** [review]

**Comment by waheedahmed** [comment]
@shafqatfarhan nice catch, done!

File: src/redesign/_style.scss (hunk 1 above)
Comment (waheedahmed) [reply src/redesign/_style.scss:535]: @shafqatfarhan it increases the height to 30px but Figma suggests 28px.

**Pull request merged**